src/extras/geometries/EdgesGeometry.js

```diff
@@ -86,6 +86,8 @@ THREE.EdgesGeometry = function ( geometry, thresholdAngle ) {
 
 	}
 
+	coords = coords.subarray( 0, index );
```

Comment:
@WestLangley Sorry, which version are you talking about? The cloned Float32Array or the temporary JS array?